tests/typ/math/matrix.typ

```diff
@@ -63,3 +63,31 @@ $ mat(B, A B) $
 $ mat(B, A B, dots) $
 $ mat(B, A B, dots;) $
 $ mat(#1, #(foo: "bar")) $
+
+---
+
+// Test matrix line drawing (augmentation).
+#grid(
+  columns: 2,
+  gutter: 10pt,
+
+  $ mat(10, 2, 3, 4; 5, 6, 7, 8; augment: #3) $,
+  $ mat(augment: #3, 10, 2, 3, 4; 5, 6, 7, 8) $,
```

Comment:
no need to test different math args parsing cases leading to the same augmentation here, argument parsing is tested separately